docs/content/connectors/oracle-cdc.md

```diff
@@ -287,6 +287,13 @@ Connector Options
       <td>Integer</td>
       <td>Integer port number of the Oracle database server.</td>
     </tr>
+  <tr>
+      <td>url</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">jdbc:oracle:thin:@{hostname}:{port}:{database-name}</td>
+      <td>String</td>
+      <td>JdbcUrl of the oracle database server .</td>
+    </tr>
```

Comment:
minor: `hostname` and `port` could be optional now

Reply:
thanks，I adjusted the contents of the document